stories/module-search-console-components.stories.js

```diff
@@ -63,8 +562,518 @@ generateReportBasedWidgetStories( {
+			],
+			position: 0,
+		},
+		{
+			clicks: 3,
+			ctr: 0,
+			impressions: 6,
+			keys: [
+				'2020-08-01',
+			],
+			position: 0,
+		},
+		{
+			clicks: 4,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-02',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-03',
+			],
+			position: 0,
+		},
+		{
+			clicks: 4,
+			ctr: 0,
+			impressions: 10,
+			keys: [
+				'2020-08-04',
+			],
+			position: 0,
+		},
+		{
+			clicks: 10,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-05',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 5,
+			keys: [
+				'2020-08-06',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-07',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 5,
+			keys: [
+				'2020-08-08',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 6,
+			keys: [
+				'2020-08-09',
+			],
+			position: 0,
+		},
+		{
+			clicks: 7,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-10',
+			],
+			position: 0,
+		},
+		{
+			clicks: 10,
+			ctr: 0,
+			impressions: 2,
+			keys: [
+				'2020-08-11',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-12',
+			],
+			position: 0,
+		},
+		{
+			clicks: 5,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-13',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-14',
+			],
+			position: 0,
+		},
+		{
+			clicks: 3,
+			ctr: 0,
+			impressions: 6,
+			keys: [
+				'2020-08-15',
+			],
+			position: 0,
+		},
+		{
+			clicks: 4,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-16',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-17',
+			],
+			position: 0,
+		},
+		{
+			clicks: 4,
+			ctr: 0,
+			impressions: 10,
+			keys: [
+				'2020-08-18',
+			],
+			position: 0,
+		},
+		{
+			clicks: 10,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-19',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 5,
+			keys: [
+				'2020-08-20',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-21',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 5,
+			keys: [
+				'2020-08-22',
+			],
+			position: 0,
+		},
+		{
+			clicks: 0,
+			ctr: 0,
+			impressions: 6,
+			keys: [
+				'2020-08-23',
+			],
+			position: 0,
+		},
+		{
+			clicks: 7,
+			ctr: 0,
+			impressions: 0,
+			keys: [
+				'2020-08-24',
+			],
+			position: 0,
+		},
+		{
+			clicks: 10,
+			ctr: 0,
+			impressions: 2,
+			keys: [
+				'2020-08-25',
+			],
+			position: 0,
+		},
+	],
+	referenceDate: '2020-08-27',
+	options: {
+		dimensions: 'date',
+		startDate: '2020-07-01',
+		endDate: '2020-08-25',
```

Comment:
I think the `url` arg is missing here - it's needed since this is the `pageDashboard` version of the widget, i.e. there is a current entity set.